tests/tests_unit/test_utils/test_time.py

```diff
@@ -35,6 +37,30 @@
     import pandas
 
 
+class TestDatetimeToIsoTimestamp:
+    @pytest.mark.skipif(platform.system() == "Windows", reason="Overriding timezone is too much hassle on Windows")
+    def test_datetime_to_iso_timestamp_timezone_unaware(self):
```

Comment:
With the class approach, you then dont need to repeat it here:
```suggestion
    def test_timezone_unaware(self):
```